nbs/15_migrate.ipynb

```diff
@@ -28,7 +28,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        slug = '/'.join(sorted(d.get('categories', '')))
</pre><p>Should this default to an empty list?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>